x/chainmain/client/cli/genaccounts.go

```diff
@@ -82,24 +82,30 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			if errstart != nil {
 				return fmt.Errorf("failed to parse vesting start: %w", errstart)
 			}
-			vestingStart, errstart := strconv.ParseInt(vestingStartStr, 10, 64)
+			vestingStartUint, errstart := strconv.ParseUint(vestingStartStr, 10, 64)
```

Comment:
should the bitsize be 63 instead of 64 (as it's later casted to a signed int64)?

Reply:
Changed to 32 to avoid overflow. Because we are converting it into `int64` later.